packages/form/src/Feedback.js

```diff
@@ -5,7 +5,17 @@ import { ErrorMessage } from 'formik';
 
 const Feedback = ({ name, ...rest }) => {
   const feedbackId = `${name}-feedback`.toLowerCase();
-  return <ErrorMessage id={feedbackId} component={FormFeedback} name={name} {...rest} />;
+  const AvFeedback = ({ children, ...rest }) => (
+    <FormFeedback {...rest}>
```

Comment:
should this be prop driven? as in, will all web applications want to show an icon in the error message? 

and if so, what if a team wanted to show `icon-attention-circle` instead of `icon-attention` ? Do we have this locked down by design to drive consistency or should this component allow for more flexibility?

Reply:
I believe we'd want it to be consistent across the portal. Teresa is the one that picked the icon.